docs/pages/application-access/jwt/grafana.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Application Access with Grafana
+description: How to configure Teleport Application Access Grafana.
+---
+
+This guide will help you configure Grafana [JWT
+authentication](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/jwt/)
+with Teleport Application Access.
+
+## Prerequisites
+
+- Teleport cluster version >= `5.0.0` with running Auth/Proxy Services or
+  Teleport Cloud account.
+- Running [Application Service](../guides/connecting-apps.mdx).
+- Grafana cluster version >= `8.4.0`.
+
+## Step 1/3. Enable a JWT authentication in Grafana
+
+Update your Grafana configuration file to enable JWT authentication. This file
+is typically located at `/usr/share/grafana/conf/defaults.ini`.
+
+```toml
+#################################### Auth JWT ##########################
+[auth.jwt]
+enabled = true
+enable_login_token = false
+header_name = Teleport-Jwt-Assertion
+email_claim =
+username_claim = username
+jwk_set_url = https://proxy.example.com:3080/.well-known/jwks.json
+jwk_set_file =
+cache_ttl = 60m
+expect_claims = {}
+key_file =
+role_attribute_path =
+role_attribute_strict = false
+auto_sign_up = true
+url_login = false
+allow_assign_grafana_admin = false
+```
+
+Let's take a closer look at the parameters and their values:
+
+- Set `enabled` to `true` to enable JWT authentication.
+- Set `header_name` to the name of the header within which the JWT assertion is
+  sent.
+- Set `username_claim` to the claim within the JWT that contains the username.
+- Set `jwk_set_url` to the address where the JSON Web Key Set (JWKS) are
+  available. Replace `proxy.example.com:3080` with the host:port where the
+  Teleport Proxy is available.
+- Set `auto_sign_up` to `true` to allow automatic creation of users upon first
```

Comment:
What roles will the auto-created user be assigned within Grafana? Maybe also mention somewhere that it will also work for pre-existing users if their Grafana username matches their Teleport username?